### PR TITLE
[zh] Localize service_mesh & service_proxy

### PR DESCRIPTION
**Pull request by Jacob953**
**Done**
- [x] add service_mesh.md localization to dev-zh.
- [x] add service_proxy.md localization to dev-zh.

### CODE REVIEW

File: content/zh/service_proxy.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## 如何帮助
+
+服务代理允许我们将这种功能“外部化”。它不再需要生活在应用程序中。
+相反，它现在被嵌入到平台层中（你的应用程序运行的地方）。
+
+作为服务之间的守门员，代理提供对正在发生的通信类型的洞察力。
+根据他们的洞察力，他们决定将一个特定的请求发送到哪里，甚至完全拒绝它。
+
+代理人收集关键数据，管理路由（在服务之间平均分配流量，或在某些服务中断时重新路由），加密连接，并缓存内容（减少资源消耗）。
```
Comment (Submarinee) [inline content/zh/service_proxy.md:25]: It's better to  leave a line here.

File: content/zh/service_mesh.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## 解决的问题
+
+在转向微服务架构后，工程师们现在要处理数百个，甚至数千个单独的服务，都需要进行通信。
+这意味着大量的流量在网络上来回流动。除此之外，单个应用程序可能需要对通信进行加密，以支持监管要求，为运营团队提供通用指标，或提供对流量的详细洞察，以帮助诊断问题。
+如果内置于单个应用程序中，这些功能中的每一个都会导致团队之间的摩擦，并减缓新功能的开发。
+
+## 如何帮助
+
+服务网格在集群的所有服务中统一增加了可靠性、可观察性和安全功能，而不需要改变代码。
+在服务网格之前，这些功能必须被编码到每一个服务中，成为错误和技术债务的潜在来源。
```
Comment (Submarinee) [inline content/zh/service_mesh.md:22]: It's better to  leave a line here.

**Review by Submarinee — APPROVED** [review]

**Comment by Jacob953** [comment]
I've done it. Thx for your review work! @Rocksnake @Submarinee

**Review by Rocksnake — APPROVED** [review]
Thanks

**Pull request merged**